src/Maestro/tests/DependencyUpdateErrorProcessor.Tests/CreateGithubIssueTests.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Maestro.Data;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace DependencyUpdateErrorProcessor.Tests
+{
+    public class CreateGithubIssueTest :DependencyUpdateErrorProcessorTests
+    {
+        [Fact]
```

Comment:
There don't appear to be any asserts in this fact... what is it supposed to be testing?

Reply:
Chad, this is just for setting up the infrastructure. I;m separating the feature flag changes with the setting up the infrastructure. I will need add the tests in the next pr.

Reply:
I see... we probably shouldn't check the [Fact] in then, since it's not really a fact.

Reply:
Moved this to setup code. I will add the test in another pr